CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## master
+
+* Adds Instabug.addExperiments API
+* Adds Instabug.removeExperiments API
+* Adds Instabug.clearAllExperiments API
```

Comment:
One changelog item would be enough for all three, trimming the names for less clutter.
```
* Adds Instabug Experiments APIs
```